Create webgpu_mrt_readback.html

Spiri0 opened the pull request:
I have an example here for the readRenderTargetAsync function. The readRenderTargetAsync is of course not intended for such a purpose, but it is nice and clear.

Attention! This example only works with the PR #28197

Spiri0 committed:
Create webgpu_rendertargets_with_gpu_readback.html

I have an example here for the readRenderTargetAsync function. The readRenderTargetAsync is of course not intended for such a purpose, but it is nice and clear.

Attention! This example only works with the PR #28197

Mugen87 commented:
Can you please rebase the PR so it refers to latest dev?

Mugen87 commented on examples/webgpu_multiple_rendertargets_with_gpu_readback.html:
```diff
@@ -0,0 +1,249 @@
+				renderer.setRenderTarget( null );
+
+				const width = renderTarget.width;
+          const height = renderTarget.height;
```
Please make sure to fix such formatting issues.

Mugen87 commented on examples/webgpu_multiple_rendertargets_with_gpu_readback.html:
```diff
@@ -0,0 +1,249 @@
+				const pixelBuffer0 = await renderer.readRenderTargetPixelsAsync(renderTarget, 0, 0, width, height, 0);	//zero is optional
+				const pixelBuffer1 = await renderer.readRenderTargetPixelsAsync(renderTarget, 0, 0, width, height, 1);
+
+				const pixelBufferTexture0 = new THREE.DataTexture(pixelBuffer0, width, height);
```
Please adhere the code style. Lines like this should look like so:
```js
const pixelBufferTexture0 = new THREE.DataTexture( pixelBuffer0, width, height );
```
Your editor should automatically fix such style issues when ESLint is proper supported.

Spiri0 committed:
Update webgpu_multiple_rendertargets_with_gpu_readback.html

formatting issue fixed

Spiri0 commented:
I fixed the formatting and also renamed the example and entered the example in the json. The topic with the rebase and the screenshot is probably only possible with the console. To do this I need to have a local github dev repository. I have to deal with that first because I haven't had much to do with it yet.

Spiri0 commented:
I'm closing this PR because I redid the  whole PR process. It was a lot of effort to do this for the first time, but it was good for understanding
